.rspec

```diff
@@ -1,3 +1,2 @@
 --color
 --require spec_helper
---order rand
```

Comment:
Can we keep the random order? (and add `--format documentation`?)

Otherwise we might be just hiding the flakyness which will resurface if we add any more specs

Reply:
No problem adding the doc format :slightly_smiling_face: For the random order, it wasn’t here before, I added with my last PR as I thought it would be a good idea, but those integration tests are quite brittle it seems :confused: Not sure how else to address this issue.